letsencrypt/main.py

```diff
@@ -32,6 +32,7 @@
 
 from letsencrypt.display import util as display_util, ops as display_ops
 from letsencrypt.plugins import disco as plugins_disco
+from letsencrypt.plugins import selection as ps
```

Comment:
Because `choose_configurator_plugins` is such a self-explanatory (but long) function name, I actually tried to import it from the module, but hit some test issues.  I guess I really dislike wrapped lines if they can be avoided :)